twolevel_dbm.py

```diff
@@ -233,6 +233,13 @@ def secondlevel(inputs, args, secondlevel=False):
                 f"antsApplyTransforms -d 3 -t [output/secondlevel/secondlevel_{subjectname}{i}0GenericAffine.mat,1] "
                 f"-r output/secondlevel/secondlevel_template0.nii.gz --verbose -o [output/compositewarps/secondlevel/{subjectname}_affine.nii.gz,1]")
 
+###############################################################################
+            # Register final model to common space
+            if args.resample_to_common_space and is_non_zero_file(args.resample_to_common_space):
```

Comment:
Can we please move the is_non_zero_file check to where the other check inputs are, so we can bail out with an error before we get to this point.